plugins/plugin-hosted/codenvy-machine-hosted/src/test/java/com/codenvy/machine/backup/MachineBackupManagerTest.java

```diff
@@ -107,15 +102,9 @@ private void setup() throws ServerException, InterruptedException, IOException,
         when(workspaceIdHashLocationFinder.calculateDirPath(any(File.class), any(String.class)))
                 .thenReturn(new File(ABSOLUTE_PATH_TO_WORKSPACE_DIR));
 
-        cmdCaptor = ArgumentCaptor.forClass(String[].class);
+        doNothing().when(backupManager).execute(anyObject(), anyInt());
 
-        doAnswer(new Answer<Void>() {
-            @Override
-            public Void answer(InvocationOnMock invocation) throws Throwable {
-                sleep(FAKE_BACKUP_TIME_MS);
-                return null;
-            }
-        }).when(backupManager).execute(anyObject(), anyInt());
+        cmdCaptor = ArgumentCaptor.forClass(String[].class);
```

Comment:
I create it in before test method just for case. It can guarantee, that each test will use its own captor object.